docs/mkdocs/docs/error_messages.md

```diff
@@ -63,7 +63,7 @@ These errors relate to data being pickled, which limits the operations available
 
 Furthermore, it is not possible to partially read/update/append the data using the ArcticDB API or use the QueryBuilder with pickled symbols. 
 
-All of these errors are of type `ArcticNativeCxxException`.
+All of these errors are of type `arcticdb_ext.exceptions.ArcticException`.
```

Comment:
A new thought just occurred: shall we create an alias for this in the `arcticdb.exception` module?

Just in case only one of `arcticdb-enterprise` and `arcticdb_ext` can be loaded.